docs/EN_US/ECLLanguageReference/ECLR_mods/WkFlo-PERSIST.xml

```diff
@@ -175,11 +177,13 @@
   <emphasis>attribute</emphasis> is used in the computation. This gives a kind
   of "compute in anticipation" capability.</para>
 
-  <para></para>
-
   <para>Example:</para>
 
-  <programlisting>  CountPeople := COUNT(Person) : PERSIST('PeopleCount');
+  <programlisting>// #OPTION ('multiplePersistInstances', true|false); // if true retains MULTIPLE, if false SINGLE
+// #OPTION ('defaultNumPersistInstances', &lt;n&gt;);      // the number to retain if MULTIPLE allowed. 
```

Comment:
minor: I'm not sure if you really want this in your example - or the option above because they're not the primary way of setting the option.

Better would be to use either ,SINGLE or ,MULTIPLE(5) on the persist.

Reply:
If the #options are included they should probably be documented in the main text.
@JamesDeFabia

Reply:
@ghalliday  I made edits, please review again 
BTW, I left the #OPTIONS  in the example because they are commented out and some people may find the info useful here. It is also in the #OPTION topic, but if all they re interested in is changing settings for PERSISTs, then this gies them the answer without having to read another topic.